code/modules/mob/pixel_shift.dm

```diff
@@ -0,0 +1,67 @@
+/mob
+	///Whether the mob is pixel shifted or not
+	var/is_shifted
```

Comment:
This should preferably be on /mob instead so it doesn't screw up vscode searching

Reply:
so var/mob/is_shifted?

Reply:
sorry am idiot nevermind

Reply:
actually cant find a place to put it.

Reply:
/code/modules/mob/mob_defines.dm